src/cli/onefuzz/backend.py

```diff
@@ -41,7 +41,8 @@
 
 _ACCESSTOKENCACHE_UMASK = 0o077
 
-ONEFUZZ_BASE_PATH = os.path.join("~", ".cache", "onefuzz")
+HOME_PATH = os.environ.get("VIRTUAL_ENV") if os.environ.get("VIRTUAL_ENV") else "~"
+ONEFUZZ_BASE_PATH = os.path.join(HOME_PATH, ".cache", "onefuzz")
```

Comment:
By modifying `ONEFUZZ_BASE_PATH`, we also modify`DEFAULT_TOKEN_PATH`. In the virtualenv case, this removes the filesystem access controls which restrict access to the token cache. We should ensure that the base path for `DEFAULT_TOKEN_PATH` stays the same (is always `$HOME`).